Source/DafnyLanguageServer.Test/DafnyLanguageServer.Test.csproj

```diff
@@ -68,6 +68,21 @@
     <None Update="Various\TestFiles\includesAincludesB.dfy">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="ProjectFiles\TestFiles\src\foo.dfy">
```

Comment:
Could we replace all these with a a single pattern somehow? I'm betting newcomers to the code base will forget to do this and be confused why their test doesn't pass.

FYI IntegrationTests does this to pick up all lit test files: https://github.com/dafny-lang/dafny/blob/master/Source/IntegrationTests/IntegrationTests.csproj#L41-L43

Understandable if this is scope creep for this PR though.

Reply:
What would be the pattern? Everything in `ProjectFiles/TestFiles` ? Then you might still forget to add another pattern for `NewFolder/TestFiles`

Reply:
I would reverse the ordering so you have `TestFile/ProjectFiles` and `TestFiles/NewFolder` instead. But again probably better not to add to this PR since it would then be a lot of edits.

Reply:
Indeed nicer to leave out of scope